lib/config.yml

```diff
@@ -350,3 +350,10 @@ Style/FloatDivision:
 
 Rails/Delegate:
   Enabled: false
+
+Layout/EmptyLinesAroundAttributeAccessor:
+  Enabled: true
```

Comment:
The default setting on this one allows certain things like `alias` but not others... which just seems arbitrary. I'm not sure it adds value.

Reply:
@avit We have a few options here (https://rubydoc.info/gems/rubocop/RuboCop/Cop/Layout/EmptyLinesAroundAttributeAccessor).  We can customize this rule with `AllowAliasSyntax` (the default), and `AllowedMethods`.

Personally, I don't mind the default.  The `alias` rule makes sense to me since they're a logical grouping.  This is mainly to ensure things like:

```
# bad
attr_accessor :foo
def do_something
end
```

Reply:
I removed this rule in https://github.com/aha-app/rubocop-aha/pull/63/commits/87ba90ab907ce823d260a80b299e119ce52c3259, since it is on by default in 0.84 and we're going to be upgrading shortly (this was originally added to get ahead of the change, but now we're behind).

Reply:
Removed after discussion, letting Rubocop set the default on this one. Let's not think about it too much.